velox/docs/configs.rst

```diff
@@ -220,7 +220,43 @@ to an existing partition.
 True if appending data to an existing unpartitioned table is allowed.
 Currently this configuration does not support appending to existing partitions.
 
-
+``Amazon S3 Configuration``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :widths: 35 10 40
+   :header-rows: 1
+
+   * - Property Name
```

Comment:
Should we have a column for "default value"?

Reply:
Not all properties require/have a default as they are optional.  So I did not add a column for this to save space.

Reply:
But to keep it consistent with other tables, I will add a default value column.

Reply:
Yes, it would be nice to have a Default Value column so readers can tell easily which properties have defaults and which are optional.